shawmut/services/autooff.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+
+import socket
+import urllib2
+import json
+import os.path
+import time
+from datetime import datetime
+from optparse import OptionParser
+from shawmut.settings import conf
```

Comment:
It is nice to have a newline between system dependencies and package dependencies.